prod/westeurope/support/redis/subnet/terragrunt.hcl

```diff
@@ -0,0 +1,19 @@
+dependency "virtual_network" {
+  config_path = "../../virtual_network"
+}
+
+# Include all settings from the root terragrunt.hcl file
+include {
+  path = find_in_parent_folders()
+}
+
+terraform {
+  source = "git::git@github.com:pagopa/io-infrastructure-modules-new.git//azurerm_subnet?ref=v2.1.28"
+}
+
+inputs = {
+  name                 = "redisbackoffice"
+  resource_group_name  = dependency.virtual_network.outputs.resource_group_name
+  virtual_network_name = dependency.virtual_network.outputs.resource_name
+  address_prefix       = "10.0.7.0/24"
```

Comment:
vnet-support address space is `10.230.1.0/24`
https://github.com/pagopa/io-infrastructure-live-new/blob/master/prod/westeurope/support/virtual_network/terragrunt.hcl#L17